projects/36/bhxiv/paper.md

```diff
@@ -293,29 +293,29 @@ To overcome this situation, two methods can be developed:
 
 
 ### Research data: ML workflows using MOWL - OWL input - output metadata
-MOWL library uses datasets in OWL format for its input data and applies machine learning methods for embedding ontologies. It generates output datasets that can further be used as features for other machine learning tasks such as classification and clustering. MOWL has already identified some generic metadata for ML and has aligned it to [Schema.org](https://schema.org/). Schema.org is a community-based project on schemas for structured data on the Internet, e.g., on web pages. MOWL metadata specification aims to cover those elements from ML datasets common to multiple domains, lowering the barrier for researchers to adopt it. The specification includes overview information (identification, name, description, keywords) provenance (creator, contributor, publisher), links to others (citation) and elements aiming to enable reusability (measurement technique, schema or standard it is based on, where it has been evaluated, distribution, size). Another example in this line, that could be adapted for the ML case, is the [computational workflow specification](https://bioschemas.org/types/ComputationalWorkflow/1.0-RELEASE) provided by [Bioschemas](https://bioschemas.org/), a community extending and tailoring Schema.org for the bioinformatics domain.
+MOWL library uses datasets in OWL format for its input data and applies ML methods for embedding ontologies. It generates output datasets that can further be used as features for other ML tasks such as classification and clustering. MOWL has already identified some generic metadata for ML and has aligned it to [Schema.org](https://schema.org/). Schema.org is a community-based project on schemas for structured data on the Internet, e.g., on web pages. MOWL metadata specification aims to cover those elements from ML datasets common to multiple domains, lowering the barrier for researchers to adopt it. The specification includes overview information (identification, name, description, keywords) provenance (creator, contributor, publisher), links to others (citation) and elements aiming to enable reusability (measurement technique, schema or standard it is based on, where it has been evaluated, distribution, size). Another example in this line, that could be adapted for the ML case, is the [computational workflow specification](https://bioschemas.org/types/ComputationalWorkflow/1.0-RELEASE) provided by [Bioschemas](https://bioschemas.org/), a community extending and tailoring Schema.org for the bioinformatics domain.
 
```

Comment:
Hi @ljgarcia I cannot see what is your change(s) suggestion here. Could you clarify? Thanks